samples/AdministrationApi/README.md

```diff
@@ -0,0 +1,57 @@
+# IIS Administration API with IIS
+
+## What is the IIS Administration API
+
+The [IIS Administration API](https://docs.microsoft.com/en-us/iis-administration/) is a Windows service that provides IIS configuration and monitoring capabilities over HTTP. The service uses REST principles and JSON payloads for manipulating and retrieving resources. With the API installed, any HTTP client can be used to edit IIS configuration, monitor performance, and browse web server files.
+
+## How This Works
+
+This repository contains a dockerfile with the steps necessary to install the IIS Administration API on the iis container image, and two configuration files that control the behavior of the API.
+
+### Configuration
+
+**api-keys.json**: This file contains hashes of the access tokens that grant access to the API. In order to communicate with the API, the tokens hash must be present in this file.
+
+**appsettings.json**: This file contains the configuration for the IIS Administration API. The biggest change from the default configuration is that Windows Authentication is disabled. This is due to no Active Directory authentication availablility within the container and the lack of pre-existing users on the image. For reference on the settings contained within the appsettings.json file visit https://docs.microsoft.com/en-us/iis-administration/configuration/appsettings.json.
+
+Before building an image from the dockerfile, both files should be configured as desired. The api-keys.json file **requires** changes while the provided appsettings.json file may be suitable for most use cases.
+
+### Adding An Api Key
+
+Before building this dockerfile, the _api-keys.json_ file should be updated with a valid api key. To acquire an api key use the _Generate-ApiKey.ps1_ PowerShell script found at https://github.com/Microsoft/IIS.Administration/tree/dev/scripts/utils. This script will generate an access token plus a token hash. The access token is a **secret** used to communicate with the API and should not be shared with anyone. Uncomment the example api key in _api-keys.json_ and update the token_hash property with the token hash value generated by the script.
+
+
+First generate an access token. The token values in this example have been removed.
+
+```
+PS C:\temp> .\Generate-ApiKey.ps1
```

Comment:
I'd rather something like this I don't have to clone to repo. 
```powershell
powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iex ((new-object net.webclient).DownloadString('htt
ps://raw.githubusercontent.com/Microsoft/IIS.Administration/dev/scripts/utils/Generate-ApiKey.ps1'))}"
```

Reply:
@shirhatti 

Great suggestion!